extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -70,6 +78,50 @@ export default class MainController implements vscode.Disposable {
 		}
 	}
 
+	/**
+	 * Creates a new SQL database project from a template, prompting the user for a name and location
+	 */
+	public async createNewProject(): Promise<void> {
+		try {
+			let newProjName = await vscode.window.showInputBox({
+				prompt: constants.newDatabaseProjectName,
+				value: `DatabaseProject${this.projectsController.projects.length + 1}`
+				// TODO: Smarter way to suggest a name.  Easy if we prompt for location first, but that feels odd...
+			});
+
+			if (!newProjName || newProjName === '') {
```

Comment:
fyi - just doing `!newProjName` will also check if it's empty (it does a falsy check which includes '' for strings`

Reply:
Good to know!